protocol-units/da-sequencer/node/src/celestia/client.rs

```diff
@@ -1,29 +1,25 @@
 use super::submit::BlobSubmitter;
-use super::{
-	BlockSource, CelestiaBlobData, CelestiaClientOps, CelestiaHeight, ExternalDaNotification,
-};
-use crate::block::SequencerBlockDigest;
+use super::{BlockSource, CelestiaBlob, CelestiaClientOps, CelestiaHeight, ExternalDaNotification};
```

Comment:
Seems like formatting isn't being applied as this is all on one line.

Reply:
Surprise! It's 98 characters long and adequately formatted.